### PR TITLE
Fix auth token context initialization

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fix token storage initialization if services are started via `[MSAppCenter startService:]` method.

## Related PRs or issues

[AB#66753](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/66753)

### CODE REVIEW

File: AppCenter/AppCenter/MSAppCenter.m [hunk 1]
```diff
@@ -416,11 +426,6 @@ - (BOOL)startService:(Class)clazz
       [service updateConfigurationWithAppSecret:appSecret transmissionTargetToken:transmissionTargetToken];
     }
 
-    // Send start service log.
-    if (sendLog && fromApplication) {
```
Comment (jwallra) [inline AppCenter/AppCenter/MSAppCenter.m]: How is this related to the auth token storage initialization?  I'm not sure why this was removed.

Comment (MatkovIvan) [reply AppCenter/AppCenter/MSAppCenter.m]: `sendLog` is called in startService**S**, the change is handle finalization logic only in one place.

The behavior of  `sendLog` isn't changed
